scss/_base_code.scss

```diff
@@ -59,7 +59,7 @@
     margin-bottom: $spv-outer--scaleable;
     margin-top: 0;
     overflow: auto;
-    padding: map-get($nudges, nudge--p-ubuntumono) $sph-inner;
+    padding: $spv-inner--x-small--scaleable $sph-inner;
```

Comment:
This changes the base styles (that we haven't updated to new style yet), is this intended here?

Reply:
Yes, this one is, @lyubomir-popov and I worked on this yesterday, it's a negligible change for the base style, but means the new snippet code blocks stay on the baseline.